nautobot_circuit_maintenance/forms.py

```diff
@@ -17,10 +16,11 @@
 from nautobot.extras.forms import (
     AddRemoveTagsForm,
     CustomFieldBulkEditForm,
-    CustomFieldFilterForm,
     CustomFieldModelForm,
     RelationshipModelForm,
```

Comment:
For completeness:

- `CustomFieldBulkEditForm` -> `nautobot.apps.forms.CustomFieldModelBulkEditFormMixin`
- `CustomFieldModelForm` -> `nautobot.apps.forms.CustomFieldModelFormMixin`
- `RelationshipModelForm` -> `nautobot.apps.forms.RelationshipModelFormMixin`